replication.go

```diff
@@ -337,6 +337,7 @@ func (r *Raft) sendLatestSnapshot(s *followerReplication) (bool, error) {
 	peer := s.peer
 	s.peerLock.RUnlock()
 
+	r.logger.Info("installing snapshot on", "peer", peer, "id", snapID, "size", req.Size)
```

Comment:
```suggestion
	r.logger.Info("installing snapshot on", "peer", peer.ID, "id", snapID, "size", req.Size)
```

Also do you mind adding `peer.ID` and `snapID` to the other two log lines in this func? It's nice when related log lines have the appropriate context to tie them all together.

Reply:
Good shout, Michael, thanks! Fixed in 9c2797b04868eff65dbfb18fae761aedd75c4c06